kong/tools/queue_schema.lua

```diff
@@ -0,0 +1,42 @@
+local Schema = require "kong.db.schema"
+
+-- TODO: enable the descriptions once they are accepted in Schemas
+return Schema.define {
+  type = "record",
+  fields = {
+    { name = {
+      type = "string",
+      -- description = "name of the queue",
+    } },
+    { batch_max_size = {
+      type = "number",
+      default = 1,
```

Comment:
So we disable batching by default? That doesn't sound like a good default.
It could be that I'm missing something.

Reply:
This is the old default.  What would be a better default?

Reply:
Maybe there should not be a default for this at all.

Reply:
Marking this as resolved for now.  A default other than 1 would be breaking, and I believe there is no good default.